Source/Eto/Forms/Controls/Splitter.cs

```diff
@@ -87,11 +89,40 @@ protected virtual void OnPositionChanged(EventArgs e)
 
 		#endregion
 
+		/// <summary>
+		/// Initializes a new instance of the <see cref="T:Eto.Forms.Splitter"/> class.
+		/// </summary>
+		public Splitter()
+		{
+			panel1MinimumSize = 0;
+			panel2MinimumSize = 0;
```

Comment:
Nah, it's all good.  This kind of pattern gets unwieldy when you have tons of members though, and it does actually generate slower code as it has to initialize the members twice.